lib/rust/text/src/text.rs

```diff
@@ -109,11 +110,16 @@ impl Text {
         }
     }
 
-    /// Constrain the selection to values valid inside of the current text buffer.
-    pub fn snap_selection(&self, selection: Selection) -> Selection {
-        let start = self.snap_location(selection.start);
-        let end = self.snap_location(selection.end);
-        selection.with_start(start).with_end(end)
+    /// Return the offset to the next codepoint if any. See the documentation of the library to
+    /// learn more about codepoints.
```

Comment:
A link to the relevant documentation might be useful for the reader.

Reply:
This refers to the ensogl_text crate's documentation. I rephrased it, and also moved the extensive docs from ensogl_text to enso_text.